lib/c0/c0_kvset.c

```diff
@@ -454,11 +454,11 @@ c0kvs_create(
         c0kvs_destroy_impl(set);
     }
 
-    cheap = cheap_create(_Alignof(max_align_t), alloc_sz);
+    cheap = cheap_create(alignof(max_align_t), alloc_sz);
     if (ev(!cheap))
         return merr(ENOMEM);
 
-    set = cheap_memalign(cheap, alignof(*set), sizeof(*set));
+    set = cheap_memalign(cheap, __alignof__(*set), sizeof(*set));
```

Comment:
Seems odd to change cheap_create from `_Alignof` to `alignof` and change cheap_memalign from `alignof` to `__alignof__`

Reply:
Good point, I'll change it to __alignof__.